src/transformers/models/paligemma/processing_paligemma.py

```diff
@@ -72,6 +74,18 @@ def build_string_from_input(prompt, bos_token, image_seq_len, image_token):
     """
     return f"{image_token * image_seq_len}{bos_token}{prompt}\n"
 
+class PaliGemmaProcessorKwargs(ProcessingKwargs, total=False):
+    _defaults = {
+        "text_kwargs": {
+            "tokenize_newline_separately": True,
+            "suffix": None,
```

Comment:
We have to add all text-related kwargs to defaults if they were used in processor's call method. We're missing `padding`, `truncation` etc. Also, `suffix` is not present in general TextKwargs, so we have to add a PaligemmaTextKwargs. See more here
https://github.com/huggingface/transformers/blob/2af199c42b545f6248475ce456dd6c2a351b8522/src/transformers/processing_utils.py#L277-L291